acdh-oeaw/vicav-vue3
# Hover tooltips

Add interactivity to HTML snippets retrieved from BASEX to support inline annotations.

Testing steps

I think you have to test this feature on the TUNOCENT dev instance, because we have no examples of this behavior in base vicav data. So I have merged the current state to the tunocent branch.

1. Go to Tounocent dev site (https://tunocent-vue.acdh-ch-dev.oeaw.ac.at/ user: tunocent pw: imaala )
2. Open linguistic features view 
Open a linguistic feature and look for a word with yellow highlight (you can try Nefza10 at the topmost part of Tunisia, they both have one near the top of the list)
Upon hover, you should see a not in a pupup box.
3. Open another linguistic feature (eg. Touila1 also in the topmost part of Tunisia) and verify if there is no interference between windows.
